.ci_support/build_all.py

```diff
@@ -207,6 +233,24 @@ def build_folders(recipes_dir, folders, arch, channel_urls):
         conda_build.api.build([recipe], config=get_config(arch, channel_urls))
 
 
+def build_folders_rattler_build(recipes_dir: str, platform, arch, channel_urls: list[str]):
+    # Local is automatically added by rattler-build so we just remove it.
+    channel_urls.remove("local")
```

Comment:
Is this list used somewhere else in the code? In case the mutation has unintended consequences, safer to rely on a copy.